desktop/frontend/src/style.css

```
@@ -0,0 +1,50 @@
+html {
+    background-color: rgba(27, 38, 54, 1);
+    text-align: center;
+    color: white;
+}
+
+body {
+    margin: 0;
+    color: white;
+    font-family: "Nunito", -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto",
```

Stylelint (17.6.0)

[error] Unexpected quotes around "Nunito" (font-family-name-quotes)

(font-family-name-quotes)